### DISCUSSION TITLE
MQTT in openHAB?

### DISCUSSION THREAD

**Discussion by DandeMC** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#89] [Q&A]
Hello, 
has someone integrated the device into openhab via mqtt? I've been trying for a while but can barely get half of it. i can display values. Unfortunately, when controlling it, it does not check the value of the device, but only remembers the one from openhab. And can't even set a temperature

**Comment by visualapproach** [comment]
Hi,
The MQTT part I can help you with. OpenHAB not at all. Maybe some one else can?

**Reply by DandeMC** [reply]
I solved it with people in the openhab forum. You can find it here if anyone need this too =)

https://community.openhab.org/t/mqtt-whirlpool-control/124283

**Reply by visualapproach** [reply]
Cool, thanks for the link.
